### PR TITLE
Adds Disabled Property to Accordion

### LINKED ISSUE #287: Add disabled property to accordion item
Hi again,
I have another feature request for the accordion: Maybe you could add a disabled property to the item so that, by setting it as true, opening that specific item would be inhibited? It would make sense to make such properties available to all components that trigger an action, actually, to keep the logic consistent.
Thank you.

### PR DESCRIPTION
**Pull request by donaldwasserman**
Fixes #287 

@simonihmig Per our ~slack~ discord conversation. Not sure if this is the approach you'd prefer for dealing with the click handler

### CODE REVIEW

**Comment by donaldwasserman** [comment]
@simonihmig Sorry, _now_ things should be passing. I forgot to run bootstrap 3 tests locally.

**Comment by donaldwasserman** [comment]
Looks like travis sort of flaked out there for some reason -- looks like the tests just need to get re-run to pass.

**Comment by simonihmig** [comment]
> Looks like travis sort of flaked out there for some reason -- looks like the tests just need to get re-run to pass.

Yeah, I always have at least one test failing for some reason (yarn registry down, browser failure, whatever). I restarted and it's green now! 🎉

Thanks again for your effort, I will review this asap, but probably not before the weekend...

File: addon/components/base/bs-accordion/item.js [hunk 1]
```diff
@@ -18,6 +18,11 @@ import layout from 'ember-bootstrap/templates/components/bs-accordion/item';
 export default Component.extend(TypeClass, {
   layout,
 
+  /**
+   * We override the mixin here to include an extra class.
+   * @see TypeClass
+   */
+  classNameBindings: ['typeClass', 'disabled'],
```
Comment (simonihmig) [inline addon/components/base/bs-accordion/item.js]: I don't think you need to add `typeClass` here, as `classNameBindings` will merge the items used in parent classes/mixins, IIRC

File: addon/components/base/bs-accordion/item/title.js [hunk 2]
```diff
@@ -23,6 +23,13 @@ export default Component.extend({
    */
   collapsed: null,
 
+  /**
+   * @property disabled
+   * @type boolean
+   * @public
```
Comment (simonihmig) [inline addon/components/base/bs-accordion/item/title.js]: Maybe mark this as `@private`, as it is only used internally, at users should set it at the `item`, not the `title`

File: addon/components/base/bs-accordion/item.js (hunk 1 above)
Comment (donaldwasserman) [reply addon/components/base/bs-accordion/item.js]: I didn't realize mixins magically combined merged those together...that's some interesting "magic"

Comment (simonihmig) [reply addon/components/base/bs-accordion/item.js]: Yeah, Ember has some places of extraordinary magic indeed! ;) 

I guess some of these are maybe going to go away some day, when moving to native classes etc. Here for the record: https://emberjs.com/api/ember/3.4/classes/EmberObject/properties/mergedProperties?anchor=mergedProperties

**Comment by simonihmig** [comment]
Thank you @donaldwasserman!

**Comment by donaldwasserman** [comment]
Thank you, @simonihmig!

**Pull request merged**